code/game/objects/items/weaponry.dm

```diff
@@ -663,7 +663,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	. = ..()
 	if(prob(1))
 		name = "cricket bat"
-		desc = "You've got red on you."
```

Comment:
Use prob(50) to select the old or the new desc. Let's just double down on code branching, I bet it'll make someone cry in the future.

Reply:
now this is a brilliant idea